delta/app/src/main/resources/app.conf

```diff
@@ -218,6 +218,8 @@ app {
     # vocab = "http://example.com/vocab"
     # base setting for the project
     # base = "http://example.com/base"
+    # to ban unconstrained resources in a project
+    enforce-schema = false
```

Comment:
I would say enforce schema is less clear because it's not obvious what aspect of the schema is enforced

Reply:
What other aspect could it be ?
In Delta, all resources can be restrained with a schema (even if it is automatic for non-generic resources)

Reply:
It might mean you are forced to create a schema in the project. It might mean that schemas must be enforced rather than give warnings

`resources-must-have-schemas`?

It's better to be verbose and clear, especially given it's used in so few places

Reply:
* Projects are a container of resources
* Schemas only apply to resources
* Schema don't have to live in the same project
* SHACL already allows levels of validation (violation, warning, ...)

Reply:
I can only say that it doesn't seem clear to me